### PR TITLE
#771 - Fix issue causing property initialisers to fail

### PR DESCRIPTION
**Pull request by treshugart**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const base = require('skatejs-build/karma.conf');
```
Comment (treshugart) [inline karma.conf.js]: Had to include the overridden Webpack config.

File: test/unit/extending.js [hunk 2]
```diff
@@ -71,7 +69,7 @@ describe('extending', () => {
     expect(el.constructor.extends).to.equal('div');
   });
 
-  if (canSetProto) {
+  if (Object.setPrototypeOf) {
```
Comment (bradleyayers) [inline test/unit/extending.js]: What is `Object.setPrototypeOf` implying?

File: webpack.config.js [hunk 3]
```diff
@@ -1 +1,2 @@
 module.exports = require('skatejs-build/webpack.config');
+module.exports.module.loaders[2].query.plugins = ['transform-class-properties'];
```
Comment (bradleyayers) [inline webpack.config.js]: This looks pretty fragile.

**Comment by bradleyayers** [comment]
lgtm

File: test/unit/extending.js (hunk 2 above)
Comment (treshugart) [inline test/unit/extending.js]: Same thing as your recently merged tests for static inheritance. I'll update to use that.

File: webpack.config.js (hunk 3 above)
Comment (treshugart) [inline webpack.config.js]: Yep it is. Do you have any suggestions? I was hoping to put this back into skatejs-build prior to this being merged, but didn't want to block if we didn't get around to it.

Comment (bradleyayers) [inline webpack.config.js]: > suggestions?

Not really. Feel free to proceed — just wanted to make sure this was a conscious decision and make you feel a bit guilty for it :P

Comment (treshugart) [inline webpack.config.js]: Lol, can always count on you ;)

Comment (bradleyayers) [inline webpack.config.js]: Sounds fine 👍

**Comment by nadiam84** [comment]
lgtm

**Pull request merged**